pilot/pkg/xds/ads.go

```diff
@@ -469,6 +484,23 @@ func (s *DiscoveryServer) initConnection(node *core.Node, con *Connection) error
 	return nil
 }
 
+func checkConnectionIdentity(con *Connection) error {
```

Comment:
maybe call `authorize`

Reply:
I changed it to this because @costinm suggested not to use authorize - I don't have a strong preference either way